tools/downloader/topologies.py

```diff
@@ -0,0 +1,233 @@
+import sys
+import os
+import re
+import yaml
+import types
+
+import common
+
+if 'INTEL_OPENVINO_DIR' in os.environ:  # As a part of OpenVINO
+    _loc = os.path.join(os.environ['INTEL_OPENVINO_DIR'], 'deployment_tools', 'open_model_zoo')
+    _modelsDir = os.path.join(_loc, 'models')
+    _downloader = os.path.join(_loc, 'tools', 'downloader', 'downloader.py')
+else:  # Standalone
+    _loc = os.path.dirname(__file__)
+    _modelsDir = os.path.join(_loc, '..', '..', 'models')
+    _downloader = os.path.join(_loc, 'downloader.py')
+
+_precisionMarker = '$precision'
+_cache = os.path.abspath(os.getenv('OPENCV_OPEN_MODEL_ZOO_CACHE_DIR', 'models'))
+
+# Parse topologies.
+for _moduleName in os.listdir(_modelsDir):
+    _modelsSubDir = os.path.join(_modelsDir, _moduleName)
+    if not os.path.isdir(_modelsSubDir):
+        continue
+
+    _module = types.ModuleType(_moduleName)
```

Comment:
I am unconvinced that each folder has to become a module, or that folders should be reflected in the API at all. The folders exist mainly to make development easier. Each model has a globally unique name, so the folder names are not required for disambiguation.

Reply:
It's a kind of open question. Do we want to have `intel` prefixes in users' apps or not to highlight which models are Intel trained and which are public.

Reply:
Removed submodules